src/text_file_functions.cpp

```diff
@@ -589,15 +590,6 @@ QStringList AOApplication::read_ini_tags(VPath p_path, QString target_tag)
   return r_values;
 }
 
-QString AOApplication::get_char_name(QString p_char)
-{
-  QString f_result = read_char_ini(p_char, "name", "Options");
-
-  if (f_result == "")
-    return p_char;
-  return f_result;
-}
-
```

Comment:
I'd like to accept this PR but `get_char_name` is still depended on according to the linker. Please bring it back or remove references to it.